src/inmanta/data/schema.py

```diff
@@ -120,6 +120,7 @@ async def _legacy_migration_table(self) -> None:
         async with self.connection.transaction():
             # lock table
             await self.connection.execute(f"LOCK TABLE {SCHEMA_VERSION_TABLE} IN ACCESS EXCLUSIVE MODE")
+            table_name_without_schema = SCHEMA_VERSION_TABLE.split(".")[1]
```

Comment:
because in the `information_schema.columns` table, the `table_name` column doesn't contain the schema neither, it's just the name. So the Where clause never matched anymore.